sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2658,8 +2666,14 @@ object Radar {
                         return
                     }
 
+                    if (event != null) {
+                        // The events are returned in the completion handler, but they're also
+                        // sent back via the RadarReceiver.
+                        receiver?.onEventsReceived(context, arrayOf(event), null)
+                    }
```

Comment:
on ios were' doing this down in `apiClient.sendEvent()`, so we should probably move this there to be consistent.